kalman_watch/low_kalman_mon.py

```diff
@@ -0,0 +1,259 @@
+"""Process telemetry and find intervals of low Kalman stars.
+
+- Intervals are stored in <data_dir>/low_kalman_events.ecsv.
+- Web dashboard is stored in <data_dir>/index_lowkals.html.
+"""
+
+import argparse
+from pathlib import Path
+
+import astropy.units as u
+import jinja2
+import numpy as np
+import plotly.graph_objects as pgo
+from astropy.table import Table, vstack
+from cxotime import CxoTime
+from Ska.engarchive import fetch
+from Ska.engarchive.utils import logical_intervals
+from ska_helpers.logging import basic_logger
+
+from . import __version__
+
+# Constants and file path definitions
+FILE_DIR = Path(__file__).parent
+
+
+def LOWKALS_DATA_PATH(data_dir: str) -> Path:
+    return Path(data_dir) / "low_kalman_events.ecsv"
```

Comment:
Agreed that this is a non-standard pattern. It comes from an historical pattern where file paths were real constants.
Now they are not but I have found that this convention makes it easier to see that the function is a file path. E.g. if you look at https://github.com/sot/kadi/blob/master/kadi/paths.py, originally in the code there was something like:
```
SKA_DATA = os.environ.get("SKA", "/proj/sot/ska")
DATA_DIR = os.environ.get("KADI", SKA_DATA)
EVENTS_DB_PATH = os.path.join(DATA_DIR, "events3.db3")
```
But then the whole paths thing got generalized and I made everything into functions but retained the naming convention.

Reply:
Given the somewhat low operational priority of this tool and the fact that it is probably not going to get looked at a lot in the future, I'd like to just leave all these upper case functions.

Going forward with new code I'll probably stop with this practice, instead putting paths into a `paths` module so that it would look like `paths.ska_data()`.